test/adapterManager.spec.js

```diff
@@ -1,402 +1,334 @@
-            ADPTMgr.registerAdapter.calledOnce.should.be.true;
-            prebid.register.calledOnce.should.be.true;
-            done();
-        });
-    });
-    /* end-test-block */
-});
+/* global describe, it, xit, sinon, expect */
+// var sinon = require("sinon");
+var should = require("chai").should();
+var expect = require("chai").expect;
+
+var ADPTMgr = require('../src_new/adapterManager');
+var CONSTANTS = require("../src_new/constants.js");
+var UTIL = require("../src_new/util.js");
+var BIDMANAGER = require("../src_new/bidManager.js");
+var prebid = require("../src_new/adapters/prebid.js");
+var conf = require('../src_new/conf');
+
+var commonAdapterID = "pubmatic";
+
+
+// TODO : remove as required during single TDD only
+// var jsdom = require('jsdom').jsdom;
+// var exposedProperties = ['window', 'navigator', 'document'];
+// global.document = jsdom('');
+// global.window = document.defaultView;
+// Object.keys(document.defaultView).forEach((property) => {
+//     if (typeof global[property] === 'undefined') {
+//         exposedProperties.push(property);
+//         global[property] = document.defaultView[property];
+//     }
+// });
+// global.navigator = {
+//     userAgent: 'node.js'
+// };
+
+describe("adapterManager : ADPTMgr", function() {
+
+
+    describe("#callAdapters", function() {
+
+        var activeSlots = null;
+
+        beforeEach(function(done) {
+            sinon.spy(UTIL, 'generateUUID');
+            sinon.spy(ADPTMgr, 'resetSlots');
+            sinon.spy(ADPTMgr, 'callAdapter');
+            activeSlots = {};
+            done();
+        });
+
+        afterEach(function(done) {
+            UTIL.generateUUID.restore();
+            ADPTMgr.resetSlots.restore();
+            ADPTMgr.callAdapter.restore();
+            activeSlots = null;
+            done();
+        });
+
+        it('is a function', function(done) {
+            ADPTMgr.callAdapters.should.be.a('function');
+            done();
+        });
+
+        it('must have called resetSlots and callAdapter internally', function(done) {
+            ADPTMgr.callAdapters(activeSlots);
+
+            UTIL.generateUUID.calledOnce.should.be.true;
+            ADPTMgr.resetSlots.calledOnce.should.be.true;
+            ADPTMgr.callAdapter.calledOnce.should.be.true;
+
+            done();
+        });
+    });    
+
+
+    /* start-test-block */
+    describe('#callAdapter', function() {
+
+        var adapters = null,
+            slots = null,
+            impressionID = null,
+            bidAdaptor = null;
+
+        beforeEach(function(done) {
+            bidAdaptor = {
+                ID: function() {
+                    // return adapterID;
+                },
+                fB: function() {
+                    return true;
+                }
+            };
+
+            sinon.spy(UTIL, 'forEachOnObject');
+            sinon.spy(prebid, "throttleAdapter");
+            sinon.spy(ADPTMgr, 'setInitTimeForSlotsForAdapter');

```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 3 tabs but found 12 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15196)